### ISSUE TITLE
User locked out if confirmation window is closed without filling.

### ISSUE THREAD

**Issue by pereiraryan** [AnomalyInnovations/serverless-stack-com#228]
Once a user signups but doesn't fill the confirmation window, they are not promoted for a confirmation textbox when trying to login again.

**Comment by jayair** [comment]
@pereiraryan Yeah we mention this in the tutorial. It is left as an exercise for the reader with some hints on how to implement it - https://serverless-stack.com/chapters/signup-with-aws-cognito.html.
